client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,1316 @@
+								checked={ termLength === 'monthly' ? 'checked' : '' }
+								name="monthly"
+								onChange={ toggleTerm }
+							/>
+							<Checkmark></Checkmark>
+							<span>{ 'Pay monthly' }</span>
+						</RadioButton>
+					</TermToggles>
+				) }
+
+				<PlanBorderOne />
+				<PlanBorderTwo featured={ true } />
+				{ selectedTab === 'Professional' && (
+					<>
+						<FeaturedPlanProfessional />
+						<PlanBorderThree />
+					</>
+				) }
+				{ selectedTab === 'Starter' && (
+					<>
+						<FeaturedPlanStarter />
+					</>
+				) }
+
+				{ planDetails &&
+					planDetails.map( ( item, index ) => (
+						<React.Fragment key={ `planDetails${ index }` }>
+							<PlanHeader
+								key={ `planHeader${ index }` }
+								className={ `tabbed-plans__header-${ index + 1 }` }
+							>
+								{ item.planName }
+								{ item.planName === 'Personal' && <span>Best to start</span> }
+								{ item.planName === 'Business' && <span>Best value</span> }
+								<p>{ bestForStrings[ item.planName ] }</p>
+							</PlanHeader>
+							<PlanPrice
+								key={ `planPrice${ index }` }
+								className={ `tabbed-plans__price-${ index + 1 }` }
+							>
+								{ formatCurrency( item.rawPrice, item.currencyCode, { stripZeros: true } ) }
+							</PlanPrice>
+							<TermDescription className={ `tabbed-plans__term-desc-${ index + 1 }` }>
+								{ item.planName === 'Free' && <span>Limited features</span> }
+								{ item.termLength === 'annually' && item.planName !== 'Free' && (
+									<>
+										<span>per month, billed annually</span>
+										<Savings>
+											{ "You're saving " +
+												Math.round(
+													( ( item.rawPriceForMonthly - item.rawPrice ) /
+														item.rawPriceForMonthly ) *
+														100
+												) +
+												'% by paying annually' }
+										</Savings>
+									</>
+								) }
+								{ termLength === 'monthly' && <span>per month, billed monthly</span> }
+							</TermDescription>
+							{ item.planName !== 'Free' && (
+								<CtaButton
+									key={ `planCtaTop${ index }` }
+									className={ `tabbed-plans__top-button-${ index + 1 }` }
+									onClick={ () => handleUpgradeButtonClick( item.planSlug, item.planProductId ) }
+									primary={ item.planName === primaryButton }
+								>
+									{ `Start with ${ item.planName }` }
+								</CtaButton>
+							) }
+							<CtaButton
+								key={ `planCta${ index }` }
+								className={ `tabbed-plans__button-${ index + 1 }` }
+								onClick={ () => handleUpgradeButtonClick( item.planSlug, item.planProductId ) }
+								primary={ item.planName === primaryButton }
+							>
+								{ `Start with ${ item.planName }` }
+							</CtaButton>
+							<SharedFeatures
+								selectedTab={ selectedTab }
+								sharedFeatures={ sharedFeatures }
+								showFeatures={ showFeatures }
+								setShowFeatures={ setShowFeatures }
+								className={ classNames(
+									'tabbed-plans__shared-features-mobile',
```

Comment:
This classname will be applied no matter it is mobile resolution or not. There is a `isMobile()` function for you to check for it if that's what you want :)
See: https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/customer-home/cards/tasks/connect-accounts/index.jsx#L1

Reply:
Great tip, this will be super helpful - thanks!